src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -204,7 +204,12 @@ def stop(self, seconds: Optional[float] = None):
             self._consumer.interrupt()
         sec_left = None if seconds is None else seconds - (time() - ts)
         self._consumer.join(sec_left)
-        self._queue.close()
+
+        # Close resources
+        self.close()
+
+        # Remove local files
+        self._queue.cleanup()
 
     def close(self):
         self._queue.close()
```

Comment:
Is the function `stop` always called before `close`? Prior to these changes, the close function used to remove files.

Reply:
> Prior to these changes, the close function used to remove files.
And `close` was called in the fork child process sometimes causing directories removal of a root process disk queue etc.